Automattic/pocket-casts-android
# Track actions that exit app from AccountDetailsFragment

## Description

This adds tracking for tapping on the privacy policy or TOS buttons on the account details screen. The reason for adding these now is because these actions "close" the app, and it would be good to have these so we know why the user is exiting the app here.

## To Test

1. Tap the profile tab
2. Tap the username bubble to open the account details
3. Tap on the Privacy Policy button
4. ✅ Observe that the event `application_details_show_privacy_policy` is tracked
5. Return to the app
6. Tap on the Terms of Service button
7. ✅ Observe that the event `application_details_show_tos` is tracked.

# Checklist
